driver-core/build.gradle

```diff
@@ -31,17 +31,17 @@ ext {
 sourceSets.main.resources.srcDirs = ['src/resources']
 
 dependencies {
-    compile project(path: ':bson', configuration: 'default')
+    api project(path: ':bson', configuration: 'default')
 
-    compile "com.github.jnr:jnr-unixsocket:$jnrUnixsocketVersion", optional
-    compile "io.netty:netty-buffer:$nettyVersion", optional
-    compile "io.netty:netty-transport:$nettyVersion", optional
-    compile "io.netty:netty-handler:$nettyVersion", optional
-    compile "org.xerial.snappy:snappy-java:$snappyVersion", optional
-    compile "com.github.luben:zstd-jni:$zstdVersion", optional
-    compile "org.mongodb:mongodb-crypt:$mongoCryptVersion", optional
+    api "com.github.jnr:jnr-unixsocket:$jnrUnixsocketVersion", optional
+    api "io.netty:netty-buffer:$nettyVersion", optional
+    api "io.netty:netty-transport:$nettyVersion", optional
+    api "io.netty:netty-handler:$nettyVersion", optional
```

Comment:
Not sure this needs to be api.

Reply:
Changing this from `api` to `implementation` will change the scope in the POM from `compile` to `runtime`. If that is an acceptable change, I will make it so.

Reply:
Yes.  That's really the point of Gradle making this change: to reduce the compilation classpath for users of the library by changing the scope in the generated pom.

Reply:
I have made the change to use `implementation`.